src/tileset.rs

```diff
@@ -53,6 +56,9 @@ pub struct Tileset {
     /// All the tiles present in this tileset, indexed by their local IDs.
     tiles: HashMap<TileId, TileData>,
 
+    /// All the wangsets present in this tileset
+    pub wangsets: Vec<WangSet>,
```

Comment:
I'd suggest to rename to `wangSets`, for consistency. Or.. I see that'd have to be `wang_sets`, so maybe we leave it be. :/

Reply:
Can't be wangSets because of standard formatting rules. wang_sets could be ok.